APIScrape/scrape.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+require_once "../Utilities/DB.php";
+
+$db = new DB();
+$dbconnection = $db->dbConnect();
+
+/**
+ * Creates a new database into which to put the scraped data.
+ * If the database already exists, tears it down and creates a new one.
+ */
+function createDatabase(PDO $db) {
+    try {
+        $sql = "DROP DATABASE IF EXISTS `cat-test`;
```

Comment:
why is it a test?

Reply:
on It.

Reply:
cat-test has been signed off as our database name.

Reply:
I'm actually not sure it has? It's not in the database design document